dataprofiler/profilers/profile_builder.py

```diff
@@ -2725,14 +2773,20 @@ def save(self, filepath: str = None) -> None:
         :type filepath: String
         :return: None
         """
+        hashing_object_dict: dict = dict()
+        if isinstance(self.hashed_row_object, HyperLogLog):
+            hashing_object_dict = self.hashed_row_object.__dict__
+        elif isinstance(self.hashed_row_object, dict):
+            hashing_object_dict = self.hashed_row_object
+
         # Create dictionary for all metadata, options, and profile
         data_dict = {
             "total_samples": self.total_samples,
             "encoding": self.encoding,
             "file_type": self.file_type,
             "row_has_null_count": self.row_has_null_count,
             "row_is_null_count": self.row_is_null_count,
-            "hashed_row_dict": self.hashed_row_dict,
```

Comment:
If this worked previously, why the change here when `hashing_object_dict = self.hashed_row_object` is the same?

Reply:
fixed